### ISSUE TITLE
Not able to start the training

### ISSUE THREAD

**Issue by SansKS7** [Event-AHU/EventVOT_Benchmark#22]
Hii author,
When i start the training by running your given command these are the result,why is it so,whats the problem?Also how to get the voxel files?

(hdetrack) root@df066691cb6f:/home/EventVOT_Benchmark/HDETrack# python tracking/train.py --script hdetrack --config hdetrack_eventvot --save_dir ./output --mode single --nproc_per_node 1 --use_wandb 0
script_name: hdetrack.py  config_name: hdetrack_eventvot.yaml
/root/miniconda3/envs/hdetrack/lib/python3.8/site-packages/timm/models/helpers.py:7: FutureWarning: Importing from timm.models.helpers is deprecated, please import via timm.models
  warnings.warn(f"Importing from {__name__} is deprecated, please import via timm.models", FutureWarning)
/root/miniconda3/envs/hdetrack/lib/python3.8/site-packages/timm/models/layers/__init__.py:48: FutureWarning: Importing from timm.models.layers is deprecated, please import via timm.layers
  warnings.warn(f"Importing from {__name__} is deprecated, please import via timm.layers", FutureWarning)
/root/miniconda3/envs/hdetrack/lib/python3.8/site-packages/timm/models/registry.py:4: FutureWarning: Importing from timm.models.registry is deprecated, please import via timm.models
  warnings.warn(f"Importing from {__name__} is deprecated, please import via timm.models", FutureWarning)
New configuration is shown below.
MODEL configuration: {'PRETRAIN_FILE': 'mae_pretrain_vit_base.pth', 'PRETRAIN_FILE_T': 'CEUTrack_ep0050.pth.tar', 'EXTRA_MERGER': False, 'RETURN_INTER': False, 'RETURN_STAGES': [], 'BACKBONE': {'TYPE': 'vit_base_patch16_224_ce', 'STRIDE': 16, 'MID_PE': False, 'SEP_SEG': False, 'CAT_MODE': 'direct', 'MERGE_LAYER': 0, 'ADD_CLS_TOKEN': False, 'CLS_TOKEN_USE_MODE': 'ignore', 'CE_LOC': [3, 6, 9], 'CE_KEEP_RATIO': [1, 1, 1], 'CE_TEMPLATE_RANGE': 'CTR_POINT'}, 'HEAD': {'TYPE': 'CENTER', 'NUM_CHANNELS': 256}}

TRAIN configuration: {'LR': 0.0004, 'WEIGHT_DECAY': 0.0001, 'EPOCH': 50, 'LR_DROP_EPOCH': 40, 'BATCH_SIZE': 1, 'NUM_WORKER': 1, 'OPTIMIZER': 'ADAMW', 'BACKBONE_MULTIPLIER': 0.1, 'GIOU_WEIGHT': 1, 'L1_WEIGHT': 14, 'FOCAL_WEIGHT': 1.0, 'FREEZE_LAYERS': [0], 'PRINT_INTERVAL': 50, 'VAL_EPOCH_INTERVAL': 50, 'GRAD_CLIP_NORM': 0.1, 'AMP': False, 'CE_START_EPOCH': 0, 'CE_WARM_EPOCH': 0, 'DROP_PATH_RATE': 0.1, 'SCHEDULER': {'TYPE': 'step', 'DECAY_RATE': 0.1, 'MILESTONES': [50, 80], 'GAMMA': 0.1}}

DATA configuration: {'SAMPLER_MODE': 'causal', 'MEAN': [0.485, 0.456, 0.406], 'STD': [0.229, 0.224, 0.225], 'MAX_SAMPLE_INTERVAL': 1, 'TRAIN': {'DATASETS_NAME': ['EventVOT'], 'DATASETS_RATIO': [1], 'SAMPLE_PER_EPOCH': 60}, 'VAL': {'DATASETS_NAME': ['EventVOT'], 'DATASETS_RATIO': [1], 'SAMPLE_PER_EPOCH': 50}, 'SEARCH': {'SIZE': 256, 'FACTOR': 4.0, 'CENTER_JITTER': 3, 'SCALE_JITTER': 0.25, 'NUMBER': 1}, 'TEMPLATE': {'NUMBER': 1, 'SIZE': 128, 'FACTOR': 2.0, 'CENTER_JITTER': 0, 'SCALE_JITTER': 0}}

TEST configuration: {'TEMPLATE_FACTOR': 2.0, 'TEMPLATE_SIZE': 128, 'SEARCH_FACTOR': 4.0, 'SEARCH_SIZE': 256, 'EPOCH': 50}

sampler_mode causal
Load pretrained model from: /home/EventVOT_Benchmark/HDETrack/lib/models/hdetrack/../../../pretrained_models/CEUTrack_ep0050.pth.tar
Load pretrained model from: /home/EventVOT_Benchmark/HDETrack/lib/models/hdetrack/../../../pretrained_models/mae_pretrain_vit_base.pth
Learnable parameters are shown below.
backbone_s.cls_token
backbone_s.pos_embed
backbone_s.pos_embed_z
backbone_s.pos_embed_x
backbone_s.pos_embed_event.pos_embedding.weight
backbone_s.pos_embed_event.pos_embedding.bias
backbone_s.pos_embed_event.proj.weight
backbone_s.pos_embed_event.proj.bias
backbone_s.patch_embed.proj.weight
backbone_s.patch_embed.proj.bias
backbone_s.blocks.0.norm1.weight
backbone_s.blocks.0.norm1.bias
backbone_s.blocks.0.attn.qkv.weight
backbone_s.blocks.0.attn.qkv.bias
backbone_s.blocks.0.attn.proj.weight
backbone_s.blocks.0.attn.proj.bias
backbone_s.blocks.0.norm2.weight
backbone_s.blocks.0.norm2.bias
backbone_s.blocks.0.mlp.fc1.weight
backbone_s.blocks.0.mlp.fc1.bias
backbone_s.blocks.0.mlp.fc2.weight
backbone_s.blocks.0.mlp.fc2.bias
backbone_s.blocks.1.norm1.weight
backbone_s.blocks.1.norm1.bias
backbone_s.blocks.1.attn.qkv.weight
backbone_s.blocks.1.attn.qkv.bias
backbone_s.blocks.1.attn.proj.weight
backbone_s.blocks.1.attn.proj.bias
backbone_s.blocks.1.norm2.weight
backbone_s.blocks.1.norm2.bias
backbone_s.blocks.1.mlp.fc1.weight
backbone_s.blocks.1.mlp.fc1.bias
backbone_s.blocks.1.mlp.fc2.weight
backbone_s.blocks.1.mlp.fc2.bias
backbone_s.blocks.2.norm1.weight
backbone_s.blocks.2.norm1.bias
backbone_s.blocks.2.attn.qkv.weight
backbone_s.blocks.2.attn.qkv.bias
backbone_s.blocks.2.attn.proj.weight
backbone_s.blocks.2.attn.proj.bias
backbone_s.blocks.2.norm2.weight
backbone_s.blocks.2.norm2.bias
backbone_s.blocks.2.mlp.fc1.weight
backbone_s.blocks.2.mlp.fc1.bias
backbone_s.blocks.2.mlp.fc2.weight
backbone_s.blocks.2.mlp.fc2.bias
backbone_s.blocks.3.norm1.weight
backbone_s.blocks.3.norm1.bias
backbone_s.blocks.3.attn.qkv.weight
backbone_s.blocks.3.attn.qkv.bias
backbone_s.blocks.3.attn.proj.weight
backbone_s.blocks.3.attn.proj.bias
backbone_s.blocks.3.norm2.weight
backbone_s.blocks.3.norm2.bias
backbone_s.blocks.3.mlp.fc1.weight
backbone_s.blocks.3.mlp.fc1.bias
backbone_s.blocks.3.mlp.fc2.weight
backbone_s.blocks.3.mlp.fc2.bias
backbone_s.blocks.4.norm1.weight
backbone_s.blocks.4.norm1.bias
backbone_s.blocks.4.attn.qkv.weight
backbone_s.blocks.4.attn.qkv.bias
backbone_s.blocks.4.attn.proj.weight
backbone_s.blocks.4.attn.proj.bias
backbone_s.blocks.4.norm2.weight
backbone_s.blocks.4.norm2.bias
backbone_s.blocks.4.mlp.fc1.weight
backbone_s.blocks.4.mlp.fc1.bias
backbone_s.blocks.4.mlp.fc2.weight
backbone_s.blocks.4.mlp.fc2.bias
backbone_s.blocks.5.norm1.weight
backbone_s.blocks.5.norm1.bias
backbone_s.blocks.5.attn.qkv.weight
backbone_s.blocks.5.attn.qkv.bias
backbone_s.blocks.5.attn.proj.weight
backbone_s.blocks.5.attn.proj.bias
backbone_s.blocks.5.norm2.weight
backbone_s.blocks.5.norm2.bias
backbone_s.blocks.5.mlp.fc1.weight
backbone_s.blocks.5.mlp.fc1.bias
backbone_s.blocks.5.mlp.fc2.weight
backbone_s.blocks.5.mlp.fc2.bias
backbone_s.blocks.6.norm1.weight
backbone_s.blocks.6.norm1.bias
backbone_s.blocks.6.attn.qkv.weight
backbone_s.blocks.6.attn.qkv.bias
backbone_s.blocks.6.attn.proj.weight
backbone_s.blocks.6.attn.proj.bias
backbone_s.blocks.6.norm2.weight
backbone_s.blocks.6.norm2.bias
backbone_s.blocks.6.mlp.fc1.weight
backbone_s.blocks.6.mlp.fc1.bias
backbone_s.blocks.6.mlp.fc2.weight
backbone_s.blocks.6.mlp.fc2.bias
backbone_s.blocks.7.norm1.weight
backbone_s.blocks.7.norm1.bias
backbone_s.blocks.7.attn.qkv.weight
backbone_s.blocks.7.attn.qkv.bias
backbone_s.blocks.7.attn.proj.weight
backbone_s.blocks.7.attn.proj.bias
backbone_s.blocks.7.norm2.weight
backbone_s.blocks.7.norm2.bias
backbone_s.blocks.7.mlp.fc1.weight
backbone_s.blocks.7.mlp.fc1.bias
backbone_s.blocks.7.mlp.fc2.weight
backbone_s.blocks.7.mlp.fc2.bias
backbone_s.blocks.8.norm1.weight
backbone_s.blocks.8.norm1.bias
backbone_s.blocks.8.attn.qkv.weight
backbone_s.blocks.8.attn.qkv.bias
backbone_s.blocks.8.attn.proj.weight
backbone_s.blocks.8.attn.proj.bias
backbone_s.blocks.8.norm2.weight
backbone_s.blocks.8.norm2.bias
backbone_s.blocks.8.mlp.fc1.weight
backbone_s.blocks.8.mlp.fc1.bias
backbone_s.blocks.8.mlp.fc2.weight
backbone_s.blocks.8.mlp.fc2.bias
backbone_s.blocks.9.norm1.weight
backbone_s.blocks.9.norm1.bias
backbone_s.blocks.9.attn.qkv.weight
backbone_s.blocks.9.attn.qkv.bias
backbone_s.blocks.9.attn.proj.weight
backbone_s.blocks.9.attn.proj.bias
backbone_s.blocks.9.norm2.weight
backbone_s.blocks.9.norm2.bias
backbone_s.blocks.9.mlp.fc1.weight
backbone_s.blocks.9.mlp.fc1.bias
backbone_s.blocks.9.mlp.fc2.weight
backbone_s.blocks.9.mlp.fc2.bias
backbone_s.blocks.10.norm1.weight
backbone_s.blocks.10.norm1.bias
backbone_s.blocks.10.attn.qkv.weight
backbone_s.blocks.10.attn.qkv.bias
backbone_s.blocks.10.attn.proj.weight
backbone_s.blocks.10.attn.proj.bias
backbone_s.blocks.10.norm2.weight
backbone_s.blocks.10.norm2.bias
backbone_s.blocks.10.mlp.fc1.weight
backbone_s.blocks.10.mlp.fc1.bias
backbone_s.blocks.10.mlp.fc2.weight
backbone_s.blocks.10.mlp.fc2.bias
backbone_s.blocks.11.norm1.weight
backbone_s.blocks.11.norm1.bias
backbone_s.blocks.11.attn.qkv.weight
backbone_s.blocks.11.attn.qkv.bias
backbone_s.blocks.11.attn.proj.weight
backbone_s.blocks.11.attn.proj.bias
backbone_s.blocks.11.norm2.weight
backbone_s.blocks.11.norm2.bias
backbone_s.blocks.11.mlp.fc1.weight
backbone_s.blocks.11.mlp.fc1.bias
backbone_s.blocks.11.mlp.fc2.weight
backbone_s.blocks.11.mlp.fc2.bias
backbone_s.norm.weight
backbone_s.norm.bias
box_head.conv1_ctr.0.weight
box_head.conv1_ctr.0.bias
box_head.conv1_ctr.1.weight
box_head.conv1_ctr.1.bias
box_head.conv2_ctr.0.weight
box_head.conv2_ctr.0.bias
box_head.conv2_ctr.1.weight
box_head.conv2_ctr.1.bias
box_head.conv3_ctr.0.weight
box_head.conv3_ctr.0.bias
box_head.conv3_ctr.1.weight
box_head.conv3_ctr.1.bias
box_head.conv4_ctr.0.weight
box_head.conv4_ctr.0.bias
box_head.conv4_ctr.1.weight
box_head.conv4_ctr.1.bias
box_head.conv5_ctr.weight
box_head.conv5_ctr.bias
box_head.conv1_offset.0.weight
box_head.conv1_offset.0.bias
box_head.conv1_offset.1.weight
box_head.conv1_offset.1.bias
box_head.conv2_offset.0.weight
box_head.conv2_offset.0.bias
box_head.conv2_offset.1.weight
box_head.conv2_offset.1.bias
box_head.conv3_offset.0.weight
box_head.conv3_offset.0.bias
box_head.conv3_offset.1.weight
box_head.conv3_offset.1.bias
box_head.conv4_offset.0.weight
box_head.conv4_offset.0.bias
box_head.conv4_offset.1.weight
box_head.conv4_offset.1.bias
box_head.conv5_offset.weight
box_head.conv5_offset.bias
box_head.conv1_size.0.weight
box_head.conv1_size.0.bias
box_head.conv1_size.1.weight
box_head.conv1_size.1.bias
box_head.conv2_size.0.weight
box_head.conv2_size.0.bias
box_head.conv2_size.1.weight
box_head.conv2_size.1.bias
box_head.conv3_size.0.weight
box_head.conv3_size.0.bias
box_head.conv3_size.1.weight
box_head.conv3_size.1.bias
box_head.conv4_size.0.weight
box_head.conv4_size.0.bias
box_head.conv4_size.1.weight
box_head.conv4_size.1.bias
box_head.conv5_size.weight
box_head.conv5_size.bias
checkpoints will be saved to /home/EventVOT_Benchmark/HDETrack/output/checkpoints
Finished training!

**Comment by wsasdsda** [comment]
@SansKS7 Hello, thanks for your attention! Based on your feedback, you may directly download the trained checkpoint and place it in the designated location so that the model will assume that you have completed the training! So if you want to retrain, please move the checkpoint away. 
In addition, if you want to use Voxel files, please refer to this code (https://github.com/Event-AHU/EventVOT_Benchmark/blob/main/Scripts/csv2voxel.py) to generate them.

**Comment by SansKS7** [comment]
okay,thanks!
